components/omega/doc/devGuide/Logging.md

```diff
@@ -0,0 +1,48 @@
+(omega-dev-logging)=
```

Comment:
This file needs to be added to `index.md`.  For now, let's just put it at the end of the list of functionality:
```
...
devGuide/Docs
devGuide/BuildDocs
devGuide/CMakeBuild
devGuide/Logging
...
```
We can reorganize in the near future.

Reply:
Added logging in index.md. The fix will be on next commit.